CLI for judges (#26)
quotient-ai/judges: README.md

- `.vote()`: Combines the judgments of multiple judges and produces a `Verdict`.

## CLI

> We provide a command-line interface for evaluating model outputs using various judges. The CLI supports both single and batch evaluations.

Reviewer: ```suggestion
`judges` also provides command-line interface (CLI) for evaluating model outputs using various judges. The CLI supports both single and batch evaluations.
```

```bash
# Basic usage
python cli.py <judge_type> <model_name> <json_input> [--out output_file]

# Example with output file
python cli.py PollMultihopCorrectness gpt-4 test_cases.json --out results.json
```

The CLI accepts the following parameters:
- `judge`: The type of judge to use (see [Classifiers](#classifiers-1))
- `model_name`: The name of the model to use (e.g., "gpt-4", "<litellm_provider>/<model_name>")

> - `json_input`: Either a JSON string or path to a JSON file containing test cases

Reviewer: Rather than having these be arguments, I think it'd be easier to know what you're running if they are parameters.

Then

```
judges PollMultihopCorrectness gpt-4 test_cases.json --out results.json
```

becomes:

```
judges PollMultihopCorrectness --model gpt-4 --input test_cases.json --output results.json
```

- `--out` (optional): Path to save the results (if not provided, prints to stdout)

Each test case in the JSON input must have:
- `input`: The input provided to the model
- `output`: The output generated by the model
- `expected`: The expected output for comparison

Example JSON input:

```json
[
    {
        "input": "What is the capital of France?",
        "output": "The capital of France is Madrid.",
        "expected": "The capital of France is Paris."
    },
    {
        "input": "What is the capital of Germany?",
        "output": "The capital of Germany is Paris.",
        "expected": "The capital of Germany is Berlin."
    }
]
```

> 
> The CLI will return a JSON object containing the original input, output, expected values, judgment score, and reasoning for each test case. It will
> either be saved in the output file or printed to std if no output file is specified.

Reviewer: ```suggestion

The CLI will return a JSON object containing the original input, output, expected values, judgment score, and reasoning for each test case. It will be saved to the output file or printed to `stdout` if no output file is specified.
```
